libc/unistd_file.c

```diff
@@ -0,0 +1,848 @@
+/*
+ * Phoenix-RTOS
+ *
+ * libc-tests
+ *
+ * Testing unistd.h file related functions
+ *
+ * Copyright 2022 Phoenix Systems
+ * Author: Mateusz Niewiadomski, Damian Loewnau
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <errno.h>
+#include <limits.h>
+#include <fcntl.h>
+#include <string.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <sys/stat.h>
+
+#include <unity_fixture.h>
+
+#define LINE1 "line1\n"
+#define LINE2 "lineline2\n" /* longer than LINE1 */
+#define LINE3 "line3\n"     /* same length as LINE1 */
+#define LINE4 "\n"
+
+#define FNAME "unistd_file_testfile"
+
+
+static int fd, fd2;
```

Comment:
It seems to me that `fd2` doesn't need to be global var

Reply:
Yeah, I think it could be a local one.